src/modules/involvementApi.js

```diff
@@ -0,0 +1,38 @@
+const BASE_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+const GAME_ID = '3bifdQ3qgzMtAvx1V3Pc';
+
+const getLikes = async () => {
+  const result = await fetch(`${BASE_URL}${GAME_ID}/likes`);
+  const data = await result.json();
+  console.log(data);
```

Comment:
- [ ] Please make sure to remove all console.logs from your codebase to keep your information safe.

Reply:
It is done thank you!